README.md

```diff
@@ -1,63 +1,37 @@
-
-## Requirements: ##
-
- - Single page application. 
- - Responsive mobile / desktop design. 
- - Use [Polymer Material Elements](https://www.webcomponents.org/author/PolymerElements) for this assignment such as:
-     - [paper-input](https://www.webcomponents.org/element/PolymerElements/paper-input).
-     - [paper-button](https://www.webcomponents.org/element/PolymerElements/paper-button).
- -  Page should closely follow [mockup](https://app.moqups.com/aaron@aarondrabeck.com/rekW1P2jz8/view).  
-	 - [Page 1 (see mock-up)](https://app.moqups.com/aaron@aarondrabeck.com/rekW1P2jz8/view/page/a2b41c7ff).
-	     - Has calculator with two states (calculator & results).
-	     - Performs calculation based on paper-input values and then switches to the result state. 
-	     - Uses this formula to calculate the amount saved: employees * (modification percent/100) * 150
-	     - Shows amount saved as a dollar amount on the results state.
-	     - The recalculate button on the results state should do these things when clicked:
-	         - a. Switch from results to the calculator state.
-	         - b. Resets the paper-input's values to 0.
-	 - [Page 2 (see-mockup)](https://app.moqups.com/aaron@aarondrabeck.com/rekW1P2jz8/view/page/a1a4f1538).
-	     - Has calculator with the same functionality as Page 1 that should now be
-   located at the top of the page. 
-	     - The amount saved from the calculator on this page should additionally be shown in the bottom text.
-	 - [Page 3 (see-mockup)](https://app.moqups.com/aaron@aarondrabeck.com/rekW1P2jz8/view/page/a9a81d7d8) 
-	     - This page should contain a simple list of the amounts saved from anytime a calculator on page one or page two is used.
-	     - On Initial Load this list should be empty.  After one calculator submit it should have one amount saved. After two calculator submits it should have two items in the list and so on.
-	     - This page should also recalculate and display the standard deviation of the amounts saved in the list whenever an item is added.
-   	     - Do not worry about persisting data on page reload.
-	      
-## Resources
-
-[Polymer](https://www.polymer-project.org/2.0/docs/devguide/feature-overview)
-
-[Web Components](https://www.webcomponents.org)
-
-[Polymer Typescript](https://github.com/LssPolymerElements/polymer2-ts)
-
-[Paper Styles](https://www.webcomponents.org/element/PolymerElements/paper-styles)
-
-[Flex Layout](https://elements.polymer-project.org/guides/flex-layout)
+------
 
+#### Acceptance Criteria ####
++ Obtains the user's full name, age, and health(poor, average, excellent).
++ Display three quotes to the user (name, premium). The quotes' premiums should be calculated from the answers to age and health.
++ Project must be able to be built on any machine with the following commands.
 
+```javascript
+	npm install
+	npm start
+```
 
+------
 
+#### Bonus Criteria ####
```

Comment:
I would replace with 

The goal of the assignment is to give you an opportunity to showcase your in-depth knowledge in modern web development.  When we evaluate your test, there are some of the things we are going to be looking at:  
+     How efficiently did you manage the use of HTML, CSS, DOM nodes?
+     How does the app respond when offline?
+     What is the TTI and Lighthouse score? Was lazy loading utilized?
+     Is the app properly componentized?
+     Is the app visually appealing, easy to use and accessible? 
+     Is the app responsive on mobile and tablets?
+     Does the app work on other browsers?
+ Where any tests written?

Reply:
I decided to remove the bonus criteria entirely. I feel like if we go down the path of telling them exactly what we want it will only become a detriment to the test. If we leave it simple and open ended it is a 2 minute read and the instructions are very clear. I still think we should look for the things you listed to set great candidates apart from the rest but we shouldn't set them as requirements. Let me know what you think!